src/Widgets/GoogleMapsWidget/GoogleMapsWidgetComponent.js

```diff
@@ -109,6 +63,20 @@ function StaticGoogleMap({ address, apiKey, zoom, children }) {
   );
 }
 
+function scaleMapSize(width, height) {
+  let newWidth = width;
+  let newHeight = height;
+
+  if (newWidth > maxWidth) {
+    newWidth = maxWidth;
+
+    const factor = height / width;
+    newHeight = Math.round(maxWidth * factor);
+  }
+
+  return { width: newWidth, height: newHeight };
```

Comment:
Now that this is a dedicated method we can simplify it further:

```suggestion
  if (width > maxWidth) {
    const factor = height / width;
    const adjustedHeight = Math.round(maxWidth * factor);

    return { width: maxWidth, height: adjustedHeight };
  }

  return { width, height };
```